internal/jimmhttp/rebac_admin/groups_test.go

```diff
@@ -28,11 +28,14 @@ import (
 func TestCreateGroup(t *testing.T) {
 	c := qt.New(t)
 	var addErr error
+	groupService := mocks.GroupService{
```

Comment:
Mock is still called GroupService, let's rename it so this doesn't end up really confusing down the line?

Reply:
Sure, to `groupMock` or `groupManagerMock`?